.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -1,86 +1,28 @@
 name: "\U0001F41B Bug Report"
 description: Report an issue or possible bug
-title: "\U0001F41B BUG:"
+title: "\U0001F41B Bug:"
 labels: ["bug"]
 
 body:
   - type: markdown
     attributes:
       value: |
-        ## Quick Bug Form
-        Thank you for taking the time to file a bug report! Please fill out this form as completely as possible.
-  - type: dropdown
-    id: products
-    attributes:
-      label: Which Cloudflare product(s) does this pertain to?
-      multiple: true
-      options:
-        - C3 (npm create cloudflare)
-        - D1
-        - KV
-        - Pages
-        - Pub/Sub
-        - Queues
-        - R2
-        - Workers for Platforms
-        - Workers AI
-        - Workers Runtime
-        - Wrangler
-        - Miniflare
-        - KV Asset Handler
-        - Workers Vitest Integration
-        - Other
-    validations:
-      required: true
+        ## Please provide as many details as possible regarding your bug.
   - type: input
     attributes:
-      label: What version(s) of the tool(s) are you using?
-      placeholder: 0.0.0 [Wrangler], 0.0.0 [C3], etc.
```

Comment:
Let's have them add the node version here, and make it optional. We aim to support all (modern) versions, and if a bug is for an odler version it's usually immediately apparent.

Reply:
Good shout collapsing the versions question into less inputs 👍